components/RepositoryList.tsx

```diff
@@ -18,10 +20,49 @@ const Loader = () => (
   </div>
 );
 
-export const RepositoryList = ({ repositories }: RepositoryListProps) => {
+export const RepositoryList = ({ repositories, dtOrder, starOrder  }: RepositoryListProps) => {
   const itemsPerScroll = 15;
   const [items, setItems] = useState(itemsPerScroll);
 
+  if (dtOrder){
+    repositories.sort(function (a, b) {
+      if (a.last_modified < b.last_modified) {
+        return 1;
+      }
+      if (a.last_modified > b.last_modified) {
+        return -1;
+      }
+      return 0;
+    });
+  }
+  
+  if(dtOrder){
+    //ordenando as Issues
+    repositories.map ((repository)=>{
+      repository.issues.sort(function (a, b) {
+        if (a.created_at < b.created_at) {
+          return 1;
+        }
+        if (a.created_at > b.created_at) {
+          return -1;
+        }
+        return 0;
+      });
+    })
+  }
+
+  if (starOrder){
```

Comment:
See comment above about having `dateOrder` and `startOrder`. It could be simplified with something like:

```
let sortFn;
switch (orderBy):
 case 'date': sortFn = (a, b) => a.last_modified < b.last_modified; break
 case 'stars': sortFn = (a, b) => a.stars < b.stars; break;
 default:  sortFn = () => (); break; // no-op
```

This could be abstracted even further by using some strategy pattern, but I doubt it'd be useful here.